.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install dependencies
+        run: npm ci
```

Comment:
We'll also need to build the TypeScript code here (`npm run build`) so that the `.js` files are present and will be included in the npm package.